### PR TITLE
chore: Refactor web.go into 3 files

### PR DESCRIPTION
**Pull request by manitgupta**
This refactors web.go into two files - `web.go`, `api/schema.go` and `api/rules.go`. Each group of APIs are put into their won file with associated UT.

This PR does not add new code, its just for refactoring.

### CODE REVIEW

File: webv2/api/rules.go [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+	session.UpdateSessionFile()
+	convm := session.ConvWithMetadata{
+		SessionMetadata: sessionState.SessionMetadata,
+		Conv:            *sessionState.Conv,
+	}
+	w.WriteHeader(http.StatusOK)
+	json.NewEncoder(w).Encode(convm)
+}
+
+// setGlobalDataType allows to change Spanner type globally.
```
Comment (asthamohta) [inline webv2/api/rules.go:231]: Should this go in rules?

Comment (manitgupta) [reply webv2/api/rules.go:231]: This actually belongs in the rules file because it is a type of a rule that can be applied via the `ApplyRule` API.

**Review by asthamohta — APPROVED** [review]

File: webv2/config.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
- "GCPProjectID": "",
- "SpannerInstanceID": ""
+ "GCPProjectID": "span-cloud-testing",
+ "SpannerInstanceID": "deep-heavy-100gb"
```
Comment (darshan-sj) [inline webv2/config.json:3]: Remove

File: webv2/api/rules.go [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+			}
+		}
+		common.ComputeNonKeyColumnSize(sessionState.Conv, tableId)
+	}
+}
+
+// addIndex checks the new name for spanner name validity, ensures the new name is already not used by existing tables
+// secondary indexes or foreign key constraints. If above checks passed then new indexes are added to the schema else appropriate
+// error thrown.
+func addIndex(newIndex ddl.CreateIndex) (ddl.CreateIndex, error) {
```
Comment (darshan-sj) [inline webv2/api/rules.go:260]: addIndex also comes under Rules?

**Review by darshan-sj — APPROVED** [review]
Looks Good!

One suggestion, In subsequent PRs, we can split api/schema.go even further. Can remove loadFromDump and dump related functions to a separate file and Conversion rate related functions to a separate file.

File: webv2/api/rules.go (hunk 3 above)
Comment (manitgupta) [reply webv2/api/rules.go:260]: Yeah, it is type of a rule.

**Pull request merged**